keps/sig-scheduling/3521-pod-scheduling-readiness/README.md

```diff
@@ -951,6 +987,9 @@ Describe them, providing:
   - Supported number of objects per namespace (for namespace-scoped objects)
 -->
 
+The feature itself doesn't generate API calls. But it's expected the API Server would receive
```

Comment:
This is an answer to the above question not this one. Please move around and answer this question (the answer is No here :) )